f5_openstack_agent/lbaasv2/drivers/bigip/l3_binding.py

```diff
@@ -115,17 +122,10 @@ def bind_address(self, subnet_id=None, ip_address=None):
                     LOG.debug('adding allowed address pair '
                               'address: %s port: %s device: %s'
                               % (ip_address, port_id, device_id))
-                    if self.driver.plugin_rpc:
-                        self.driver.plugin_rpc.add_allowed_address(
-                            port_id=port_id,
-                            ip_address=ip_address
-                        )
-                    else:
-                        LOG.error(
-                            'No RPC to plugin available to add '
-                            'allowed address %s to port: %s.'
-                            % (ip_address, port_id)
-                        )
+                    self.driver.plugin_rpc.add_allowed_address(
```

Comment:
It's not clear from the commit message why this change is necessary.  You've remove some error handling.  Do we need a try/except around this call- even if the old if.self.driver.plugin.rpc check is no longer necessary?